src/polyfem/assembler/RhsAssembler.hpp

```diff
@@ -120,9 +120,9 @@ namespace polyfem
 			const mesh::Mesh &mesh_;
 			const mesh::Obstacle &obstacle_;
 			const int n_basis_;
-			const int size_;
-			const std::vector<basis::ElementBases> &bases_;
-			const std::vector<basis::ElementBases> &gbases_;
+			const int size_; //< dimension of problem
```

Comment:
You should use three slashes (`///<`) so it is picked up by Doxygen.

Reply:
Got it, thanks. I can add that. I also see that a couple files have /// @brief above the class description. Do I want to add that as well?

Reply:
I forget if it is required for a single-line comment, but it is for multiline comments.

Reply:
Okay, that makes sense.

I just updated the other comments to use the triple slash